packages/eslint-config-finn-prettier/react.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+    extends: [
+        require.resolve('./prettier.js'),
+        'prettier/react',
```

Comment:
only on clashing rules (of which there are none). Configs are merged recursively